1.3.4 Orientation: update guidance for non-web documents and software and add examples (#780)
w3c/wcag2ict: comments-by-guideline-and-success-criterion.md

This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.3](https://www.w3.org/WAI/WCAG22/Understanding/sensory-characteristics#intent).

##### orientation
###### Applying SC 1.3.4 Orientation to Non-Web Documents

This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.4](https://www.w3.org/WAI/WCAG22/Understanding/orientation#intent).

> 

Author: ```suggestion
<div class="note wcag2ict documents">

Content that is only used on hardware with a fixed display orientation where a user cannot change the orientation is covered under the essential exception and does not need to provide support for orientation changes.</div>
```

##### orientation
###### Applying SC 1.3.4 Orientation to Non-Web Documents

Where the non-web document is displayed on hardware that is designed to be reoriented in typical use, this applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.4](https://www.w3.org/WAI/WCAG22/Understanding/orientation#intent).
<div class="note wcag2ict documents">

A non-web document that is only used on hardware that supports a single display orientation, or is only displayed on hardware that is physically fixed in one orientation (e.g. a digital building directory) is excluded by the precondition and therefore does not need to provide support for orientation changes.</div>

###### Applying SC 1.3.4 Orientation to Non-Web Software

Where the non-web software is displayed on hardware that is designed to be reoriented in typical use, this applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.4](https://www.w3.org/WAI/WCAG22/Understanding/orientation#intent).

<div class="note wcag2ict software">
    

> Content that is only used on hardware with a fixed display orientation where a user cannot change the orientation is covered under the essential exception and does not need to provide support for orientation changes.</div>

Author: ```suggestion
Non-web software that is only run on hardware that supports a single display orientation, or is run only on hardware that is physically fixed in one orientation (e.g. a digital building directory) is excluded by the precondition and therefore does not need to provide support for orientation changes.</div>
```

This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.3](https://www.w3.org/WAI/WCAG22/Understanding/sensory-characteristics#intent).

##### orientation
###### Applying SC 1.3.4 Orientation to Non-Web Documents

This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.4](https://www.w3.org/WAI/WCAG22/Understanding/orientation#intent).
<div class="note wcag2ict documents">

Content that is only used on hardware with a fixed display orientation where a user cannot change the orientation is covered under the essential exception and does not need to provide support for orientation changes.</div>

> <div class="note wcag2ict documents">

Author: ```suggestion
<div class="example wcag2ict documents">
```
